components/gb-live/live.web.tsx

```diff
@@ -32,11 +33,18 @@ const NativeAppDownloadLinks: React.FC = () => {
 };
 
 export const GbLive = () => {
-  useGbLive();
+  const refetch = useGbLive();
   return (
     <WithTermsAndConditionsAccepted>
       <>
-        <NativeAppDownloadLinks />
+        <WithSkiaWeb
+          // getComponent={() => import("./svg-map/skia-test")}
+          getComponent={async () => {
+            const SvgMap = (await import("./svg-map/svg-map")).default;
+            return { default: () => <SvgMap refetch={refetch} /> };
+          }}
+          fallback={<span>Loading Map...</span>}
```

Comment:
NEEEEEDS to be fixed, probably to a giant blue square, because while refetching the screen flashes between the map and this and it's awful